OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -79,17 +79,15 @@ public MusicPlaylist(World world, MusicPlaylistInfo info)
 				currentSong = currentBackgroundSong = world.Map.Rules.Music[info.BackgroundMusic];
 				CurrentSongIsBackground = true;
 			}
-			else
-			{
-				// Start playback with a random song
-				currentSong = random.FirstOrDefault();
```

Comment:
This disables the fallback to a random song when no background music is set. It's not documented and not used anywhere so I thought this can go away.

Reply:
Just because someone disables music shuffling does not mean to penalize him/her with all matches starting with Await and Bigfoot. The randomization is in place to shake that up.

Reply:
Ok, I just changed this because the description says it would start with the first song, which should then be adjusted. I like the idea that the game always starts with the same song unless the player the changes it, but I will change it if requested.